### DISCUSSION TITLE
Meshnet

### DISCUSSION THREAD

**Discussion by Cheekie25** [bubuntux/nordlynx#119] [Q&A]
Hello,
Is it possible to use Mesnet functionnality with nordlynx ?

**Comment by mofongox** [comment]
I just installed NordLynx in my unraid, and while it is connected to a Nord IP, I can't seem to make it work for meshnet (which is the only reason why I want this in my Unraid).

I would also like to know how to enable meshnet with:
Allow Incoming Traffic: enabled
Allow Routing: enabled
Allow Local Network Access: enabled
Allow Sending Files: enabled
Accept Fileshare Automatically: enabled

Thank you beforehand.
